### DISCUSSION TITLE
Migrating from 4.4.1 to 5.1.0: how to use existing H2 database?

### DISCUSSION THREAD

**Discussion by saniac** [Athou/commafeed#1541] [Q&A] [answered]
I have tried to run 5.1.0 against my existing data directory (which was created with 4.4.1, using the default H2 database).

When I start commafeed I get this error: "Datasource '<default>': Wrong user name or password [28000-232]"

And this is followed by a stack trace
[stacktrace.txt](https://github.com/user-attachments/files/16859314/stacktrace.txt)

I have tried passing in the user name and password from the old config.yml on the command line like this, but it doesn't work:

./commafeed-5.1.0-h2-linux-x86_64-runner -Dquarkus.datasource.username=sa:quarkus.database.password=REDACTED

What am I doing wrong?

**Comment by Athou** [comment]
II think it should work out of the box, did you change the default password in your `config.yml` file?

If so, I don't think you can separate properties with a `:` on the command line, did you find this somewhere?
Please try with `./commafeed-5.1.0-h2-linux-x86_64-runner -Dquarkus.datasource.username=sa -Dquarkus.database.password=REDACTED`

**Comment by saniac** [comment, marked as answer]
Thank you, I have solved my problem(s).

You are right, I did change the default password.

What was worked was `./commafeed-5.1.0-h2-linux-x86_64-runner -Dquarkus.datasource.username=sa -Dquarkus.datasource.password=REDACTED` 

I now have a working, migrated database.

I realise now beyond not separating arguments correctly, I mistyped the name of the property for the password (it is quarkus.datasource.password not quarkus.database.password).

Thanks again for your help.
